src/modules/logger.ts

```diff
@@ -47,12 +47,12 @@ export const logger = winston.createLogger({
 		format.timestamp({ format: "YYYY-MM-DD HH:mm:ss" }),
 		format.printf(info => `${info.timestamp} ${info.level}: ${info.message}` + (info.splat !== undefined ? `${info.splat}` : " "))
 	),
-	transports: [
+	transports: process.env.DEVELOPMENT ? undefined : [
```

Comment:
Probably want to check process.env.DEVELOPMENT === "true" because it can also be set to "false" which would steal equal to true